snuba/settings/__init__.py

```diff
@@ -128,6 +128,8 @@
 # Migrations in skipped groups will not be run
 SKIPPED_MIGRATION_GROUPS: Set[str] = {"metrics", "querylog", "spans_experimental"}
 
+SNQL_SUBSCRIPTION_ROLLOUT = 0.0
```

Comment:
How does this work if I need to turn SnQL subscriptions off suddenly? Do I need to deploy a change or is there a faster way?

Reply:
Use redis backed config flags that can be turned on/off at runtime 

https://github.com/getsentry/snuba/blob/master/snuba/state/__init__.py#L162-L174